tests/validation/tests/v3_api/test_auth.py

```diff
@@ -741,21 +741,24 @@ def create_project_client(request):
 
     cluster_total = len(client.list_cluster().data)
     node_roles = [["controlplane", "etcd", "worker"]]
+    kargs = {'node_roles': node_roles,
+             'random_cluster_name': True,
+             'validate': False}
     aws_nodes1 = cluster1 = None
     aws_nodes2 = cluster2 = None
     if cluster_total == 0:
-        cluster1, aws_nodes1 = create_and_validate_custom_host(node_roles, True)
-        cluster2, aws_nodes2 = create_and_validate_custom_host(node_roles, True)
+        cluster1, aws_nodes1 = create_and_validate_custom_host(**kargs)
+        cluster2, aws_nodes2 = create_and_validate_custom_host(**kargs)
     if cluster_total == 1:
-        cluster1, aws_nodes1 = create_and_validate_custom_host(node_roles, True)
+        cluster2, aws_nodes2 = create_and_validate_custom_host(**kargs)
```

Comment:
Is it possible that cluster_total > 1 ?

Reply:
Yes, if you prepare the requirements yourself and execute the `needs_cluster` jenkins job then there wouldn't be need for creating clusters. There's a check for this further down in the code where the test asserts `clusters >= 2`